module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -509,6 +520,16 @@ public function init()
         $this->sessionCache = $factory($namespace);
     }
 
+    protected function getInnReachDb()
```

Comment:
This function will need a doc comment to avoid style complaints. Maybe something like:

```
/**
 * Establish INN-Reach database connection.
 *
 * @return void
 */
```